src/_gettsim/transfers/kinderzuschlag/regelbedarf.py

```diff
@@ -0,0 +1,170 @@
+"""Basic needs of children in the calculation of Kinderzuschlag."""
+
+from _gettsim.functions.policy_function import policy_function
+
+
+@policy_function()
+def kosten_unterkunft_m_bg(
+    wohnbedarf_anteil_eltern_bg: float,
+    arbeitslosengeld_2__bruttokaltmiete_m_bg: float,
+    arbeitslosengeld_2__heizkosten_m_bg: float,
+) -> float:
+    """Calculate costs of living eligible to claim.
+
+    Unlike ALG2, there is no check on whether living costs are "appropriate".
+
+    Parameters
+    ----------
+    wohnbedarf_anteil_eltern_bg
+        See :func:`wohnbedarf_anteil_eltern_bg`.
+    arbeitslosengeld_2__bruttokaltmiete_m_bg
+        See :func:`arbeitslosengeld_2__bruttokaltmiete_m_bg`.
+    arbeitslosengeld_2__heizkosten_m_bg
+        See :func:`arbeitslosengeld_2__heizkosten_m_bg`.
+
+    Returns
+    -------
+
+    """
+    warmmiete_m_bg = (
+        arbeitslosengeld_2__bruttokaltmiete_m_bg + arbeitslosengeld_2__heizkosten_m_bg
+    )
+
+    out = wohnbedarf_anteil_eltern_bg * warmmiete_m_bg
+
+    return out
+
+
+@policy_function()
+def wohnbedarf_anteil_eltern_bg(
+    anzahl_kinder_bg: int,
+    demographic_vars__anzahl_erwachsene_bg: int,
+    kinderzuschl_params: dict,
+) -> float:
+    """Calculate living needs broken down to the parents. Defined as parents'
+    subsistence level on housing, divided by sum of subsistence level from parents and
+    children.
+
+    Reference: § 6a Abs. 5 S. 3 BKGG
+
+    Parameters
+    ----------
+    anzahl_kinder_bg
+        See :func:`anzahl_kinder_bg`.
+    demographic_vars__anzahl_erwachsene_bg
+        See :func:`demographic_vars__anzahl_erwachsene_bg`.
+    kinderzuschl_params
+        See params documentation :ref:`kinderzuschl_params <kinderzuschl_params>`.
+
+    Returns
+    -------
+
+    """
+    ex_min = kinderzuschl_params["existenzminimum"]
+
+    # Up to 10 children are considered
+    considered_children = min(anzahl_kinder_bg, 10)
+    single_oder_paar = (
+        "single" if demographic_vars__anzahl_erwachsene_bg == 1 else "paare"
+    )
+
+    out = (
+        ex_min["kosten_der_unterkunft"][single_oder_paar]
+        + ex_min["heizkosten"][single_oder_paar]
+    ) / (
+        ex_min["kosten_der_unterkunft"][single_oder_paar]
+        + ex_min["heizkosten"][single_oder_paar]
+        + (
+            considered_children
+            * (
+                ex_min["kosten_der_unterkunft"]["kinder"]
+                + ex_min["heizkosten"]["kinder"]
+            )
+        )
+    )
+
+    return out
+
+
+@policy_function(end_date="2010-12-31", leaf_name="regelsatz_m_bg")
+def regelsatz_m_bg_arbeitsl_geld_2_params_old(
```

Comment:
```suggestion
def regelsatz_m_bg_arbeitsl_geld_2_params_bis_2010(
```

But doesn't it make sense to calculate this in `arbeitslosengeld_2` and use it from there? I guess over there, we immediately add `kosten_der_unterkunft` and that's why we don't have it? 

*but then below, we do exactly that. what is the difference? We even use the `arbeitsl_geld_2_params` here*

Reply:
KdU are different for ALG2 and KiZ. But the Regelsatz is the same, I created #826.

However, the name `regelbedarf` for the module and `bedarf` for the target is confusing. While it is technically correct, it is only about the Bedarf of the parents and only enters `anzurechnendes_einkommen`. I will merge this module with the `einkommen` module because much of the code should go anyhow. That should avoid further confusion.